src/main/resources/db/migration/V2__Create_band_table.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE band (
+    bandId INT PRIMARY KEY,
+    bandName VARCHAR(255) NOT NULL
```

Comment:
also too long varchar, I would limit it to 25 chars

Reply:
Fixed